service/application/src/main/java/com/solace/maas/ep/event/management/agent/command/CommandManager.java

```diff
@@ -83,14 +92,19 @@ public void configPush(CommandRequest request) {
             finalizeAndSendResponse(request);
             return;
         }
+        List<Path> listOfExecutionLogFiles = new ArrayList<>();
 
         for (CommandBundle bundle : request.getCommandBundles()) {
             // For now everything is run serially
             for (Command command : bundle.getCommands()) {
+                Path executionLog = null;
                 try {
                     switch (command.getCommandType()) {
                         case terraform:
-                            terraformManager.execute(request, command, envVars);
+                            executionLog = terraformManager.execute(request, command, envVars);
+                            if (listOfExecutionLogFiles != null) {
```

Comment:
The variable is initialized above so it will never be null. Can we just remove this check? It would help with the complexity metric as well.

Reply:
If I am not wrong but the TerraformManager `MAY` return a null path : https://github.com/SolaceProducts/event-management-agent/pull/174/files#diff-8dda03e5a7dc0775f19837dd861b625895276658ab058153b5f7cc008913c0feR53 
and 
https://github.com/SolaceProducts/event-management-agent/pull/174/files#diff-8dda03e5a7dc0775f19837dd861b625895276658ab058153b5f7cc008913c0feR57-R63

Reply:
This is a typo, it should be `if(executionLog != null)`. I will update the code and add some test for the scenario

Reply:
Updated the code